apps/judicial-system/web/src/routes/Shared/Cases/ActiveCases.tsx

```diff
@@ -72,12 +66,12 @@ const ActiveCases: React.FC<React.PropsWithChildren<Props>> = (props) => {
       direction: 'descending',
     },
   )
-  const [displayCases, setDisplayCases] = useState<CaseListEntry[]>([])
-  const [modalVisible, setVisibleModal] = useState<'DELETE_CASE'>()
-  // The index of requset that's about to be removed
-  const [requestToRemoveIndex, setRequestToRemoveIndex] = useState<number>(-1)
   const { isOpeningCaseId, showLoading, handleOpenCase, LoadingIndicator } =
     useCaseList()
+  const [displayCases, setDisplayCases] = useState<CaseListEntry[]>([])
+  const [modalVisible, setVisibleModal] = useState<'DELETE_CASE'>()
+  // The id of the case that's about to be removed
+  const [caseToRemove, setCaseToRemove] = useState<CaseListEntry | undefined>()
```

Comment:
If the parameter list in useState is empty, the type automatically becomes `T | undefined`, so there is no need to specify CaseListEntry | undefined here.